tests/Unit/Biilink/RequestFactory/SessionCreateRequest/OrderItemFactoryTest.php

```diff
@@ -213,6 +215,53 @@ public function test_it_returns_product_tax_amount_zero_if_order_item_tax_total_
         $this->assertSame(0.0, $orderItem->productTaxAmount);
     }
 
+    public function test_it_creates_order_item_for_shipment(): void
+    {
+        $shippingMethod = $this->prophesize(ShippingMethodInterface::class);
+        $shippingMethod->getCode()->willReturn('DHL');
+        $shippingMethod->getName()->willReturn('DHL Shipping');
+        $shippingMethod->getDescription()->willReturn('Ship with DHL');
+
+        $shipment = $this->prophesize(ShipmentInterface::class);
+        $shipment->getMethod()->willReturn($shippingMethod);
+        $shipment->getShippingUnitTotal()->willReturn(1234, 1234);
+        $shipment->getAdjustmentsTotal()->willReturn(567);
+
+        $factory = $this->createTestSubject();
+        $shipmentItem = $factory->createForShipment($shipment->reveal());
+
+        self::assertSame('DHL', $shipmentItem->productCode);
```

Comment:
```suggestion
        $this->assertSame('DHL', $shipmentItem->productCode);
```
and apply it everywhere.